src/ui/components/NetworkMonitor/NetworkMonitorListRow.module.css

```diff
@@ -138,7 +135,6 @@
 }
 
 .Row[data-selected] .TimingContainer,
-.Row[data-selected]:focus .TimingContainer,
```

Comment:
These styles seemed super broken (still are on production) and they aren't necessary. We already style the selected row differently anyway.